buildspec.yml

```diff
@@ -1,7 +1,65 @@
 version: 0.2
 
 phases:
+  install:
+    runtime-versions:
+      golang: 1.12
+  pre_build:
+    commands:
+      # GOPATH is setup like the following in Codebuild standard 2.0 image
+      # /go:/codebuild/output/src<some numbers>
+      # so we copy all the source code to the appropriate location before
+      # invoking any go command
+      - echo "cd into $CODEBUILD_SRC_DIR"
+      - cd $CODEBUILD_SRC_DIR
+      - mkdir -p /go/src/github.com/aws/amazon-ecs-cli
+      - cp -R ./* /go/src/github.com/aws/amazon-ecs-cli/
+      - ls -lah /go/src
+      - |
+        env -i PATH=$PATH GOPATH=`go env GOPATH` GOROOT=`go env GOROOT` GOCACHE=`go env GOCACHE` \
+        go test -race -v -cover github.com/aws/amazon-ecs-cli/ecs-cli/modules/...
   build:
     commands:
-      - make integ-test
-      - ./bin/local/ecs-cli.test
\ No newline at end of file
+      - echo "Compilation context:"
+      - echo "CODEBUILD_SOURCE_VERSION=$CODEBUILD_SOURCE_VERSION"
+      # set the version number to the tag on HEAD or "development" if HEAD is
+      # not tagged
+      - VERSION=`git tag --points-at HEAD`
+      - |
+        if [ -z "$VERSION" ]; then
+          VERSION='development'
+        fi
```

Comment:
Similarly -- should we take the same approach as we do in [v2](https://github.com/aws/amazon-ecs-cli-v2/pull/212/files#diff-ab45564d4a3a96a4af2f6455a2b3613eR9)?